cmd/operator/main.go

```diff
@@ -77,7 +77,7 @@ func main() {
 
 	signalHandler := ctrl.SetupSignalHandler()
 
-	startWebhookAndBootstrapperIfDebugFlagSet(startupInfo{
+	startWebhookIfDebugFlagSet(startupInfo{
 		cfg:           cfg,
```

Comment:
In addition to remove the bootstrapper from the function name we should remove the function `startBootstrapperManager` too as the bootstrapper no longer exists

Reply:
removed